include/ips4o/bucket_pointers.hpp

```diff
@@ -60,7 +60,9 @@ class Sorter<Cfg>::BucketPointers {
 
 #elif defined(__SIZEOF_INT128__)
 
-    using atomic_type = unsigned __int128;
+    // __extension__ suppresses the pedantic warning about __int128
+    // not being ISO C++ compliant
+    __extension__ using atomic_type = unsigned __int128;
```

Comment:
Unless I'm mistaken, you can also use `__uint128_t` to avoid the pedantic warning without having to use `__extension__`.

Reply:
It seems that it does, and a quick Google suggests that it supported by at least Clang, GCC, and Intel's compiler. I don't know enough about compilers and support to say for certain that this is a safe replacement, but if it is then it might be preferable to `__extension__`.

Reply:
I'm unfortunately no more knowledgeable, I just know that the use of `__int128_t` and `__uint128_t` is somewhat widespread and have the advantage of being "normal" from the language POV, just another type which happens to have compiler support.